Importers/Ifc/Speckle.Importers.Ifc/Converters/SpatialStructureConverter.cs

```diff
@@ -1,31 +1,52 @@
 ﻿using Speckle.Importers.Ifc.Ara3D.IfcParser.Schema;
 using Speckle.Importers.Ifc.Types;
 using Speckle.InterfaceGenerator;
+using Speckle.Objects.Data;
+using Speckle.Sdk.Models;
 using Speckle.Sdk.Models.Collections;
 
 namespace Speckle.Importers.Ifc.Converters;
 
 [GenerateAutoInterface]
-public sealed class IfcSpatialStructureElementConverter : IIfcSpatialStructureElementConverter
+public sealed class IfcSpatialStructureElementConverter(IGeometryConverter geometryConverter)
+  : IIfcSpatialStructureElementConverter
 {
   public Collection Convert(IfcModel model, IfcSpatialStructureElement node, INodeConverter childrenConverter)
   {
-    if (!node.IsIfcRoot) //I'd really rather have a class for this (IfcRoot : IfcNode)
-      throw new ArgumentException("Expected to be an IfcRoot", paramName: nameof(node));
+    var directGeometry = ConvertAsDataObject(model, node);
 
+    var relationalChildren = childrenConverter.ConvertChildren(model, node);
+    var allChildren = relationalChildren.Prepend(directGeometry).ToList();
+
+    //We're preferring to keep IFC collections lightweight, and adding a DataObject with the properties
+    // 1. Spatial elements can can have direct geometry (mostly only common with IFC Site)
+    // 2. Keeps property access simpler
     return new Collection
     {
-      name = node.Name ?? node.Guid,
-      applicationId = node.Guid,
-      elements = childrenConverter.ConvertChildren(model, node),
+      name = node.Name ?? node.LongName ?? node.Guid,
+      elements = allChildren,
+      ["expressID"] = node.Id,
```

Comment:
would even remove this to prevent two objects having the same express id, but not a blocker